js/samples/quickstart-csharp/QuickstartSampleWebApp.csproj

```diff
@@ -1,14 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <UserSecretsId>c857223c-72cb-4ebe-92a7-8e06a34f87df</UserSecretsId>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
```

Comment:
didn't we remove this from other sample conversions earlier?

<Nullable>enable</Nullable>

Reply:
yes, removed too